### DISCUSSION TITLE
Overlayscrollbars inside of iframe

### DISCUSSION THREAD

**Discussion by isaakdc** [KingSora/OverlayScrollbars#687] [Q&A]
I would like to use overlayscrollbars inside of an iframe embeded in a React application. Is there any way of doing this? I've tried rendering the <OverlayScrollbarsComponent> as a iframe element, but this doesn't seem to work.

```
<OverlayScrollbarsComponent
    className="z-20 size-full"
    defer
    element="iframe"
    id="order-preview"
    name="order-preview"
></OverlayScrollbarsComponent>
```

**Comment by KingSora** [comment]
Good day @isaakdc :)

The problem here is, that an `iframe`s content is completely isolated from the rest of your website, thus there are challenges to overcome. Are you embedding an 3rd party website in the iframe or is it a website owned by you?

**Reply by michael-koch787** [reply]
Is it possible?
The website inside the iframe is owned by me.

**Reply by KingSora** [reply]
Sure then just initialize overlayscrollbars on the page inside the iframe

**Reply by michael-koch787** [reply]
Makes sense. Thanks.
And what about if I don't own the website or using TinyMCE with iframe mode?
